### ISSUE TITLE
Dialogue commands require spaces

### ISSUE THREAD

**Issue by PascalLadalle** [4ian/GDevelop#1613]
In the new dialogue tree features, the _command_ feature requires a space before it (or after it, depending on where you place it). Otherwise, it creates problems like skipping a character or skipping a trigger.
![image](https://user-images.githubusercontent.com/25978830/78186584-410f4b00-746d-11ea-9f9a-a9aeb7b988e8.png)
On this screenshot, for instance, if I don't add a space before the command, the next line displays next to the one from the previous branch (which should be deleted by the scrolling effect).

**Comment by PascalLadalle** [comment]
![image](https://user-images.githubusercontent.com/25978830/78191270-f3e3a700-7475-11ea-9530-98d26d87a885.png)
Here, if I remove the space, it crashes the scrolling (only the first character of the line above appears).

**Comment by blurymind** [comment]
Can you share the file please

**Comment by PascalLadalle** [comment]
Using the dialog tree example, I added a tag to the first branch.
As a result, the dialog doesn't show automatically, I need to press z. And the 2nd line displays next to the first one without waiting for the z trigger.
![image](https://user-images.githubusercontent.com/25978830/78272491-de21c080-750d-11ea-846a-6196b99c9f97.png)
In some occasions, adding a space fixes the issue, but here it doesn't.

**Comment by blurymind** [comment]
Again, I need you to zip up a minimal example game project and attach it to this issue. :)

**Comment by PascalLadalle** [comment]
Here goes. 
[Dialogue tree.zip](https://github.com/4ian/GDevelop/files/4423221/Dialogue.tree.zip)

**Comment by blurymind** [comment]
Btw what you have there is a command, not a tag

**Comment by PascalLadalle** [comment]
Thanks for the reply. You're right, I mixed up the _lingo_, sorry.
Actually, it's not just skipping a space, it can also skip a trigger depending on where it is, which is more annoying. Or it "skips" the automatic showing of the dialogue (and key instructions), which is even more annoying.
Should I close this bug report since the problem is external, or leave it open while waiting for the external fix? I don't know much about the github _decorum_

**Comment by blurymind** [comment]
if its actually skipping to execute some` <<command>>`, that is then something that has to be fixed. Does your example contain that issue?
If not, upload a new one

**Comment by PascalLadalle** [comment]
What I mean is that in the default example, the dialogue shows automatically when the game has loaded.
https://editor.gdevelop-app.com/?project=example://dialogue-tree-with-yarn
In the project I shared, it no longer does it, you have to trigger it by typing z.

**Comment by blurymind** [comment]
In the example there is an event in the sheet I think that triggers it

**Comment by PascalLadalle** [comment]
Yes, when the player drops to the floor next to the sign at the beginning, the dialogue is supposed to be triggered without needing a "z".
![image](https://user-images.githubusercontent.com/25978830/78510679-6d8dd480-7797-11ea-8330-9728ef7adcd4.png)

**Comment by PascalLadalle** [comment]
@blurymind Testing on the example, if I place the _avatar_ command at the end of the first line, it skips the _wait for trigger_ , if I place it at the beginning of the second line, the same happens, and if I place it at the end of the second line, it breaks the dialogue (second line cannot display).

**Comment by blurymind** [comment]
I'm trying to fix this atm. There is a or that adds a stress test

**Comment by PascalLadalle** [comment]
Two years later, [bumping this issue](https://forum.gdevelop.io/t/solved-problem-with-yarn-variables/38983).
